DigitalLearningSolutions.Data.Tests/Services/TutorialContentServiceTests.cs

```diff
@@ -0,0 +1,70 @@
+﻿namespace DigitalLearningSolutions.Data.Tests.Services
+{
+    using DigitalLearningSolutions.Data.Models.TutorialContent;
+    using DigitalLearningSolutions.Data.Services;
+    using DigitalLearningSolutions.Data.Tests.Helpers;
+    using FluentAssertions;
+    using NUnit.Framework;
+
+    internal class TutorialContentServiceTests
+    {
+        private TutorialContentService tutorialContentService;
+
+        [SetUp]
+        public void Setup()
+        {
+            var connection = ServiceTestHelper.GetDatabaseConnection();
+            tutorialContentService = new TutorialContentService(connection);
+        }
+
+        [Test]
+        public void Get_tutorial_content_of_consistent_tutorial_should_return_tutorial()

```

Comment:
What do you mean by "consistent"? 🙂

Reply:
The tutorial ID exists on that customisation and section, and there is an `aspProgress` entry for this tutorial ID and candidate ID. Maybe this test could be renamed to explain this? I can't think of any concise ways of saying this, do you have an suggestions? 🙂

Reply:
Maybe having a few more tests:
* `Get_tutorial_content_should_return_tutorial()`
* `Get_tutorial_content_should_return_null_if_tutorial_id_invalid()`
* Same as above but customisation id invalid
* Same as above but section id invalid

What do you think? 🙂

Reply:
I like it! Done in bd57872 🙂